app/admin/[organization]/page.tsx

```diff
@@ -14,26 +15,23 @@ export async function generateStaticParams() {
 }
 
 const EventPage = async ({ params }: { params: { organization: string } }) => {
-  const events: IEvent[] = await (
-    await fetch(`${apiUrl()}/organizations/${params.organization}/events`, {
```

Comment:
not sure we should need / switch to Axios. Nextjs uses the default fetch WebAPI is shouldn't really need any additional dependencies.

Reply:
I see Axios was already installed actually. Made an issue to remove axios completely, but I'd recommend to at least switch back to fetch here.
https://github.com/streamethorg/streameth-platform/issues/79